src/app/components/lineage-graph/lineage-graph.component.ts

```diff
@@ -51,16 +47,12 @@ export class LineageGraphComponent implements OnChanges, OnInit {
     public view: [number, number] = [this.INITIAL_GRAPH_VIEW_WIDTH, this.INITIAL_GRAPH_VIEW_HEIGHT];
     public showSidePanel = false;
 
-    @ViewChild("containerRef", { static: true }) element: ElementRef;
+    @ViewChild("containerRef", { static: false }) element: ElementRef;
```

Comment:
Can you explain this change?

Reply:
When I used true, I get error in console.
`false` - the result will be obtained only after passing the first change check.